src/normalizer.ts

```diff
@@ -76,10 +76,22 @@ export type InternalShape = ObjectBase & {
 
 export type InternalSlideObject = InternalText | InternalImage | InternalShape;
 
+export type InternalPathSrc = {
+  path: string;
+}
+
+export type InternalDataSrc = {
+  data: string;
+}
+
+export type InternalImageSrc = (InternalPathSrc | InternalDataSrc ) & {
```

Comment:
With `{kind: "data", data: string} | {kind: "path", path: string}` this will have a bit better type checking

Reply:
Cool, good to know, tx!